src/CMakeLists.txt

```diff
@@ -535,6 +535,12 @@ else()
     OUTPUT_NAME leancpp)
 endif()
 
+if((${STAGE} GREATER 0) AND CADICAL)
```

Comment:
It copies `cadical` into (in stage1) `build/release/stage1/cadical` which is next to `build/release/stage1/lean`. This is necessary because `bv_decide` looks for `cadical` next to `lean` and when we run tests our `lean` binary is still in that directory and not in whatever directory `install` copies `cadical` to.